### ISSUE TITLE
php7 config parse fail

### ISSUE THREAD

**Issue by waplet** [sensiolabs/SensioDistributionBundle#288]
```
php -l vendor/sensio/distribution-bundle/Resources/skeleton/web/config.php
PHP Parse error:  syntax error, unexpected '&&' (T_BOOLEAN_AND) in vendor/sensio/distribution-bundle/Resources/skeleton/web/config.php on line 200
```
**config.php**
```
<?php if (!$hasMajorProblems) && !$hasMinorProblems): ?>
    <p class="ok">All checks passed successfully. Your system is ready to run Symfony applications.</p>
<?php endif; ?>
```

Do you Merge without syntax checks? :D

https://github.com/sensiolabs/SensioDistributionBundle/blob/master/Resources/skeleton/web/config.php#L200

Edit: similar error in line 205 - https://github.com/sensiolabs/SensioDistributionBundle/blob/master/Resources/skeleton/web/config.php#L205
**PHP**:
```
PHP 7.0.10-2+deb.sury.org~trusty+1 (cli) ( NTS )
Copyright (c) 1997-2016 The PHP Group
Zend Engine v3.0.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.10-2+deb.sury.org~trusty+1, Copyright (c) 1999-2016, by Zend Technologies

```

Oh, there is already PR - https://github.com/sensiolabs/SensioDistributionBundle/pull/285/files

**Comment by waplet** [comment]
https://github.com/sensiolabs/SensioDistributionBundle/blob/master/Resources/skeleton/web/config.php#L200

**Comment by HeahDude** [comment]
Will be closed by #285.
